cmd/tailscale/backend.go

```diff
@@ -55,19 +58,28 @@ const (
 	loginMethodWeb    = "web"
 )
 
-var fallbackNameservers = []netaddr.IP{netaddr.IPv4(8, 8, 8, 8), netaddr.IPv4(8, 8, 4, 4)}
+// googleDnsServers are used on ChromeOS, where an empty VpnBuilder DNS setting results
+// in erasing the platform DNS servers. The developer docs say this is not supposed to happen,
+// but nonetheless it does.
+var googleDnsServers = []netaddr.IP{netaddr.IPv4(8, 8, 8, 8), netaddr.IPv4(8, 8, 4, 4),
+	netaddr.IPv6Raw([16]byte{0x20, 0x01, 0x48, 0x60, 0x48, 0x60, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x88, 0x88}),
```

Comment:
I'd just use netaddr.MustParseIP for all 4 of these ... it's super fast, doesn't allocate, etc. And it's much more readable.

Reply:
Done.